src/Parser/DeathKnight/Frost/Modules/Features/HardHowlingBlastCasts.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import Analyzer from 'Parser/Core/Analyzer';
+import SPELLS from 'common/SPELLS';
+import SpellIcon from 'common/SpellIcon';
+import AbilityTracker from 'Parser/Core/Modules/AbilityTracker';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+import Combatants from 'Parser/Core/Modules/Combatants';
+
+class HardHowlingBlastCasts extends Analyzer {
+  static dependencies = {
+    abilityTracker: AbilityTracker,
+    combatants: Combatants,
+  };
+
+  rimeprocsCounter = 0;
+  rimedHBCounter = 0;
```

Comment:
"HB" can be omitted as the class is already called "HowlingBlast". How about calling this `castsWithRime`?